.github/RELEASE.md

```diff
@@ -0,0 +1,177 @@
+The complete LibreTime documentation is available at libretime.org.
+
+The full tarball for the 3.0.0-alpha.8 release of LibreTime is available here.
+
+Since this is an alpha release there will be bugs in the code.
+
+Please report new issues and/or feature requests in the issue tracker. Join our discourse or chat us up on Slack if you need help and for general discussion.
+
+## Table of Contents
+
+    Features
+    Bugfixes
+    Deprecated Features
+    Contributors
+    Installation
+    Updating
+    Known Issues
+        Installer Issues
+            Media-Monitor config needs manual removing
+            Outdated silan reports unreliable cue in/out information
+        Liquidsoap support
+        No watched folder support
+        No Line In recording support
+        Playout wont work if locale is missing
+        Lack of i18n toolchain is disturbing
+
+## Features
+
+    The LibreTime project now has a proper logo!
+    New "Show Listener Stats" in "Analytics" contains listeners statistics on specific shows.
+    Display time of last podcast import in downloaded podcasts view.
+    Allow sorting by last play date in smartblocks, makes creating playlists that play the least played track possible.
+    Preliminary support for Debian Buster (Remember to patch the liquidsoap scripts if you use Buster).
+
+## Bugfixes
+
+    Widgets now use user specified timezones as they should.
+    Podcast view now display proper number of downloaded podcasts rather than just the first 25 podcasts.
+    Fix using non-ascii characters in podcast publishing service.
+    Fix canceling current show for "linked" shows.
+    Fix empty schedule page when previous track is empty.
+    Fix focus jumping to search in advanced search.
+
+## Deprecated Features
+
+    We are phasing out support for Debian 8 (Jessie). See the announcement in the 3.0.0-alpha.7 release notes for more details.
+    Liquidsoap 1.1.1 support. 3.0.0-alpha.8 is most likely the last version to support liquidsoap 1.1.1 out of the box. The number of distros that install a current version of liquidsoap are gaining the majority and once Ubuntu releases a version of their distro that has liquidsoap 1.3.x we will switch to native liquidsoap 1.3.x support. Users still on liquidsoap 1.1.1 will need to apply a patch to their liquidsoap scripts (or update liquidsoap).
+
+## Contributors
+
+The LibreTime project wants to thank the following contributors for adding PRs to this release:
+
+    @ciaby
```

Comment:
Maybe this list can be automated?

Reply:
Specifically the contributors?
If automated in general, we could look into something like this: https://github.com/marketplace/actions/release-notes-generator.

Reply:
For the moment, I suggest we leave it as is and can look at https://github.com/marketplace/actions/release-notes-generator later